src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -593,11 +612,56 @@ def docKeyUpdate(self, msOutDoc, **kwargs):
                 self.logger.warning(msg)
         return msOutDoc
 
-    def docInfoUpdate(self, msOutDoc):
+    def docInfoUpdate(self, msOutDoc, pipeLine=None):
```

Comment:
this new argument seems to be used only for the exception code. How about we print the whole document in the exception (where one can see whether it was a RelVal or standard workflow.

Reply:
The document is already fully logged in debug  mode at the end of the consumer pipeline..... Which is not exactly relevant to the current message, but simply pointing out that there are already way to many times one and the same document will be fully dumped in the log. I am more for actually making our way of handling exceptions from inside the Pipeline so we can log them properly and in a consistent manner or at least to go through the whole code and make all the messages follow the same schema, like: [1]. I lately find it quite difficult to relate some of the old messages that are not following [1] to the proper module pipeline and workflow. 
[1]
```
"%s: %s: Some message regarding msOutDoc with '_id': %s" % (self.currThreadIdent, pipelineName, msOutDoc['_id'])
```